docs/xdr/features/collect/ingestion_methods/sekoiaio_docker_concentrator.md

```diff
@@ -0,0 +1,435 @@
+
+Two files are needed to run the concentrator: `docker-compose.yml` and `intakes.yaml`.
+
+1. Create a folder where the configuration and data of the concentrator will be stored
+
+```bash
+mkdir sekoiaio-concentrator && cd sekoiaio-concentrator
+```
+
+2. Create the two files
+
+```bash
+touch docker-compose.yml && touch intakes.yaml
+```
+
+### Configure intakes.yaml file
+
+The `intakes.yaml` file is used to tell the concentrator how to bind a port where logs are received to its technology represented by the Intake key.
+For each technology, specify:
+
+* a name: it has nothing to do with SEKOIA.IO, feel free to use the explicite value of your choice
+* the protocol: `tcp` or `udp`
+* a port: to process incoming events
+* the Intake key: can be retreived from the Intakes page of your community
+
+Here is an example of the file with 3 technologies sending their events to the concentrator on ports `20516`, `20517` and `20518`. Feel free to copy paste it and adapt it to your need:
+
+```yaml
+---
+intakes:
+- name: Techno1
+  protocol: tcp
+  port: 20516
+  intake_key: INTAKE_KEY_FOR_TECHNO_1
+- name: Techno2
+  protocol: udp
+  port: 20517
+  intake_key: INTAKE_KEY_FOR_TECHNO_2
+- name: Techno3
+  protocol: tcp
+  port: 20518
+  intake_key: INTAKE_KEY_FOR_TECHNO_3
+```
+
+!!! Tip
+    You are not limited to 3 entries. Feel free to adapt it to your needs.
+
+### Configure docker-compose.yml file
+
+Please find below a template of the `docker-compose.yml` file.
+
+```yaml
+version: "3.9"
+services:
+  sekoia_concentrator:
+    logging:
+      options:
+        max-size: "1000m"
+        max-file: "2"
+    image: ghcr.io/sekoia-io/sekoiaio-docker-concentrator:0.9
+    environment:
+      - MEMORY_MESSAGES=100000
+      - DISK_SPACE=32g
+    ports:
+      - "20516-20518:20516-20518"
+    volumes:
+      - ./intakes.yaml:/intakes.yaml
+      - ./conf:/etc/rsyslog.d
+      - ./disk_queue:/var/spool/rsyslog
+    restart: always
+    pull_policy: always
+```
+
+!!! Note
+    Feel free to copy paste it and adapt it to your needs.
+
+The following sections will describe each element of this file above.
+
+#### Logging
+
+```yaml
+logging:
+    options:
+    max-size: "1000m"
+    max-file: "2"
+```
+
+Docker logging system offers the flexibility to view events received on the container in real time with the command `docker logs <container_name>`. These logs are stored by default in `/var/lib/docker/containers/<container_uuid>/<container_uuid>-json.log`. To avoid the overload of disk space on your host, some options are specified. `max-size` specifies the maximum size of one file and `max-file` specifies the total number of files allowed. When the maximum number of files is reached, a log rotation is performed and the oldest file is deleted.
+
+!!! Note
+    Docker logging system is an independent solution from SEKOIA.IO IO or the buffer you want to set up on your concentrator. It is only used to view the last events on your concentrator.

```

Comment:
Docker logging system is an independent solution from SEKOIA.IO or the buffer you want to set up on your concentrator. It is only used to view the last events on your concentrator.